test/cppsuite/tests/run.cxx

```diff
@@ -152,6 +155,14 @@ main(int argc, char *argv[])
             print_help();
             return 0;
         } else if (std::string(argv[i]) == "-C") {
+            if ((i + 1) < argc)
+                wt_open_config = argv[++i];
+                /* Add a comma to the front if the user didn't supply one. */
```

Comment:
This doesn't seem like the right spot to do this. Can we do it when we're putting together the config string in `test::run`?

Reply:
Sure